### PR TITLE
create rows and cells classes

### PR DESCRIPTION
**Pull request by sivanisentinel**

### CODE REVIEW

**Comment by Nitzperetz** [comment]
@sivanisentinel , looks good!
Please resolve conflicts by merging from master.
Also, seems like there would be lint issues, run `npm run check` in order to make there are no errors.

File: src/components/Layouts/ReportLayout.js [hunk 1]
```diff
@@ -24,13 +24,17 @@ const ReportLayout = ({ sections, headerLeftImage, headerRightImage }) => {
         Object
           .keys(sections)
           .map((rowPos) =>
-            <div className="report-row" key={rowPos} style={sections[rowPos].style}>
+            <div
+              className={ sections[rowPos][0].class ? 'report-row ' +sections[rowPos][0].class : 'report-row'}
```
Comment (Nitzperetz) [inline src/components/Layouts/ReportLayout.js]: Just to be on the safe side, can you make sure `sections[rowPos]` is not empty before addressing the first index?

Comment (sivanisentinel) [reply src/components/Layouts/ReportLayout.js]: can we have a row without any columns? I didn't se any option to define rows without at least one column

Comment (Nitzperetz) [reply src/components/Layouts/ReportLayout.js]: This is fine then.
You will still have lint errors, I would fix it and merge.

Comment (sivanisentinel) [reply src/components/Layouts/ReportLayout.js]: Thanks!

**Pull request merged**